Lombiq.Privacy.Samples/Manifest.cs

```diff
@@ -0,0 +1,19 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "Lombiq Privacy - Samples",
+    Author = "Lombiq Technologies",
+    Website = "https://github.com/Lombiq/Orchard-Privacy",
+    Version = "3.1.0-alpha",
+    Description = "Samples for Lombiq Privacy.",
+    Category = "Privacy",
+    Dependencies = new[]
+    {
+        "Lombiq.HelpfulExtensions.ContentTypes",
```

Comment:
Why is this hard coded? If your project depends on it, then it should be referenced, otherwise it won't become a dependency in the nuget package.